plugin/pkg/scheduler/factory/factory.go

```diff
@@ -131,13 +131,7 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys util.StringSe
 
 	// Watch minions.
 	// Minions may be listed frequently, so provide a local up-to-date cache.
-	if false {
```

Comment:
Actually, I think reflector will not work but poller will work.  We don't yet support list by fields now.

The reason you see pods are recreated back to unhealthy node is due to #5399.